steward/src/cellars/cellar_v2_5.rs

```diff
@@ -274,6 +274,26 @@ pub fn get_encoded_function(call: FunctionCall, cellar_id: String) -> Result<Vec
 
             Ok(CellarV2_5Calls::SetSharePriceOracle(call).encode())
         }
+        Function::CachePriceRouter(params) => {
+            validate_cache_price_router(
+                &cellar_id,
+                params.check_total_assets,
+                params.allowable_range,
+                Some(&params.expected_price_router),
```

Comment:
We should be normalizing the input we're receiving either here or in the validation function, since we are comparing expected price routers against hardcoded normalized values.